Documentation/ListModule/UsingEffectively/Index.rst

```diff
@@ -23,56 +23,56 @@ Select the :guilabel:`Web > List` module and browse to the
 .. include:: /Images/AutomaticScreenshots/ListModule/ListModule.rst.txt
 
 The list view displays a list of records stored on the
-current page, grouped by type. The name which appears for each record
-depends on which field is used as label. For "pages", the field is
+current page, grouped by type. The name which appears on a record
+depends on which field is used for the label. For "pages", the field is
 "Pagetitle".
 
-The list view offers several different action icons in the **Docheader**:
+The list view has action icons in the **Docheader** including:
 
 1. **Create** a new record (a wizard appears to let you choose which type)
 
 2. **View** the current page (in the frontend)
 
 3. **Edit** the current page
 
-4. **Search** in the current page
+4. **Search** the current page
 
 5. **Clear the cache** of the current page
 
 6. **Refresh** the List view
 
 7. **Bookmark** this view
 
-8. **Copy URL** to this view
+8. **Copy URL** of this view
 
 9. Access **contextual help** (removed with version 12.0)
 
 .. todo: update screenshot
 
-In the **Content Area**, action icons are also available:
+Action icons are also available in the **Content Area**:
 
 .. include:: /Images/AutomaticScreenshots/ListModule/ListModuleContentArea.rst.txt
```

Comment:
Could you provide an updated screenshot showing TYPO3 v12 maybe?

Reply:
I will take care of updating the screenshots in this manual.

Reply:
I'll just do this one. I'm trying to learn Gimp :-)

Reply:
Tried to draw an arrow with Gimp. Had to install a plugin (how do I install a plugin?) After that: Where it that plugin, and then it hasn't worked. Switched to Photoshop Elements -> done in 3 minutes (including starting the Windows VM).

Good luck.